### PR TITLE
[IMP] website: introduce `s_discovery` snippet

### PR DESCRIPTION
**Pull request by chgo-odoo**
This PR introduces the new `s_discovery` snippet.

| Snippet in use |
|--------|
| image | 
---------

### Checklist
- [x] Meaningful demo-text
- [x] Adapts to color combinations -> Readable text
- [x] Mobile friendly
- [x] Has no `<h1>` tags (except for "Intro" snippets)
- [x] Has one `<h1>` tag ("Intro" snippets only)
- [x] Images are optimized
- [x] Images can be replaced by the website wizard (with fairly apparent exceptions)
- [x] No ineffective options. Eg "click -> nothing happens"
- [x] Edition is limited when not necessary
- [x] Behaves consistently with the rest of the snippets (images can be replaced, etc..)

task-4077581
Part of task-4077427

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by chgo-odoo** [comment]
@stefanorigano Hello, this one is ready for review 👍 

I have temporarily included the `s_cta_badge` within this PR so you can see the expect visual review, of course it'll be removed after `s_cta_badge` is merged.

Could you have a look, please?

Thanks !

**Review by stefanorigano — COMMENTED** [review]
Hello @chgo-odoo , seems ok :)
Let's wait for`s_cta_badge` to be merged

File: addons/website/views/snippets/s_discovery.xml
Comment (Brieuc-brd) [inline addons/website/views/snippets/s_discovery.xml:5]: Is `o_cc o_cc1` really necessary by default ?

File: addons/website/views/snippets/s_discovery.xml
Comment (Brieuc-brd) [inline addons/website/views/snippets/s_discovery.xml:8]: I know it doesn't seem logical, but if you want to define "1x" value in the WB by default you should remove `fa-1x` 😅 (I guess this should also be fixed in `cta_badge` 😉 )
Capture d’écran 2024-08-22 à 11 04 31
```suggestion
                <i class="fa fa-fw fa-info-circle o_not-animable" role="img"/>&#160;&#160;What do you want to promote&amp;nbsp;? &#160;&#160;&#160;&#160;<a href="#">See more <i class="fa fa-long-arrow-right" role="img"/></a>
```

**Review by Brieuc-brd — CHANGES_REQUESTED** [review]
Thanks @chgo-odoo 👍 
I've left a few comments ;)

File: addons/website/views/snippets/s_discovery.xml
Comment (chgo-odoo) [reply addons/website/views/snippets/s_discovery.xml:8]: Not related to the `s_cta_badge` only but there's indeed an issue 😅 I'll fix this occurrences and will open a bugfix PR to fix this behaviour in `s_badge` and `s_cta_badge` 👍

**Review by Brieuc-brd — APPROVED** [review]
LGTM, good job 👍

**Comment by bso-odoo** [comment]
LGTM - queued after `s_sidegrid` #175616

**Comment by bso-odoo** [comment]
@robodoo delegate=bvr-odoo

**Comment by bvr-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
